modules/custom/d_commerce_products_list/src/Plugin/Block/CommerceResetButton.php

```diff
@@ -15,7 +18,55 @@
  *   category = @Translation("Facets"),
  * )
  */
-class CommerceResetButton extends BlockBase {
+class CommerceResetButton extends BlockBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * Request.
```

Comment:
Ja bym chyba zmienił opis tu i w konstruktorze, bo request stack a request to nie to samo ;-)
ew. od razu ustawiał w konstruktorze `$this->request = $requestStack->getCurrentRequest()->getRequestUri()`
bo tylko po to za pewne Ci potrzebny był request stack.